packages/python/test/classdeclarations.test.tsx

```diff
@@ -316,21 +315,20 @@ describe("Python Class - FunctionDeclaration", () => {
               type={{ children: "int" }}
               instanceVariable
             />
-            <py.FunctionDeclaration
+            <py.MethodDeclaration
               name="my_method"
               parameters={[
                 { name: "a", type: { children: "int" } },
                 { name: "b", type: { children: "int" } },
               ]}
               returnType={{ children: "int" }}
-              instanceFunction
               refkey={methodRefkey}
             >
               return a + b
-            </py.FunctionDeclaration>
+            </py.MethodDeclaration>
             <py.FunctionDeclaration
               name="my_class_method"
-              classFunction
+              functionType="class"
               returnType={{ children: "int" }}
               refkey={classMethodRefkey}
             >
```

Comment:
See my comment above as to whether we want to allow `FunctionDeclaration` use with `functionType`.